scripts/submit_bb.py

```diff
@@ -19,9 +20,8 @@
 default_wct = "00:30:00"
 
 
-def main(args):
+def main(args, est_model=None):
```

Comment:
for consistency, change this to what submit_emod3d.py has?
stricter but more clear of declaration of usage of variables.

Reply:
Done